core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -1793,7 +1797,7 @@ static class BitOpImplementor extends StrictAggImplementor {
       if (SqlTypeUtil.isBinary(info.returnRelType())) {
         start = Expressions.field(null, ByteString.class, "EMPTY");
       } else {
-        Object initValue = info.aggregation() == BIT_AND ? -1L : 0;
+        Object initValue = info.aggregation().kind == SqlKind.BIT_AND ? -1L : 0;
```

Comment:
This change is just to be consistent with the other functions right?

Reply:
This change is to make sure "BITAND_AGG" and "BIT_AND" are caught, previously it was just checking for a specific operator so "BITAND_AGG" was not going down the correct codepath